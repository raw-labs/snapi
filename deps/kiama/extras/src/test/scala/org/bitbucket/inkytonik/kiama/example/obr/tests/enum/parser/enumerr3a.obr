(* Missing comma between enumeration constant identifiers *)

PROGRAM Err803a (i : INTEGER) : INTEGER;

VAR
    x : ( harold bob );

BEGIN
END Err803a.
