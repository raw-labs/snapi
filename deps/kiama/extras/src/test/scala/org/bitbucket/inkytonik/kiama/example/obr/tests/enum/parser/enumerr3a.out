src/test/scala/org/bitbucket/inkytonik/kiama/example/obr/tests/enum/parser/enumerr3a.obr:6:18:error: ')' expected but 'b' found
    x : ( harold bob );
                 ^
